### PR TITLE
configure obsReport.NewExporter with struct

### LINKED ISSUE #2643: Change `NewExporterObsReport` to accept `Settings/Params` instead of direct arguments

### PR DESCRIPTION
**Pull request by naseemkullah**
fixes #2643

### CODE REVIEW

File: obsreport/obsreport_exporter.go [hunk 1]
```diff
@@ -98,7 +98,13 @@ type Exporter struct {
 	mutators     []tag.Mutator
 }
 
-func NewExporter(level configtelemetry.Level, exporterName string) *Exporter {
+type ExporterConfig struct {
```
Comment (bogdandrutu) [inline obsreport/obsreport_exporter.go:101]: s/ExporterConfig/ExporterSettings

Comment (naseemkullah) [reply obsreport/obsreport_exporter.go]: ✅

**Comment by bogdandrutu** [comment]
this needs a rebase :)

**Comment by naseemkullah** [comment]
> this needs a rebase :)

done!

**Review by bogdandrutu — APPROVED** [review]

**Pull request merged**